src/views/temp.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
please check if we can get rid of this file

Reply:
let it be for a moment...will delete later